specs/electra/beacon-chain.md

```diff
@@ -258,36 +255,27 @@ class PendingConsolidation(Container):
     target_index: ValidatorIndex
 ```
 
-### Modified Containers
+#### `ExecutionLayerRequests`
 
-#### `AttesterSlashing`
+*Note*: This container holds requests from the execution layer that are received in [
+`ExecutionPayloadV4`](https://github.com/ethereum/execution-apis/blob/main/src/engine/prague.md#executionpayloadv4) via
+the Engine API. These requests are required for CL state transition (see `BeaconBlockBody`).
 
 ```python
-class AttesterSlashing(Container):
-    attestation_1: IndexedAttestation  # [Modified in Electra:EIP7549]
-    attestation_2: IndexedAttestation  # [Modified in Electra:EIP7549]
+class ExecutionLayerRequests(Container):
```

Comment:
Personally, I would prefer the name `ExecutionRequests` because:

* It mirrors `ExecutionPayload`.
* We use the field name `execution_requests` in `BeaconBlockBody`.
* And it's shorter.

Reply:
I'm happy to rename it to `ExecutionPayload`. I'll just wait until others weight in before renaming it in the PR.